sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -20,6 +20,13 @@
                 "description": "Friendly name."
             }
         },
+        "kind": {
```

Comment:
Do we want to extend this template, or create a separate one for featurestore workspace? @seanyao1, can you advice?

Reply:
I think this approach makes sense here given featurestore is "kind" of workspace. Featurestore is wrapper around workspace using workspace operations. It probably makes sense to use the workspace template since that is what workspace operations are using. @seanyao1 to weigh in as well.